colossalai/inference/config.py

```diff
@@ -111,6 +111,7 @@ class InferenceConfig:
         use_cuda_graph (bool): Whether to enforce CUDA graph execution. If False, we will disable CUDA graph and always execute the model in eager mode. If True, we will use eager execution in hybrid.
         max_context_len_to_capture (int): max context len that could be captured by CUDA Graph, per sequence
         high_precision(Optional[bool]): Whether to use float32 for underlying calculations of float16 data to achieve higher precision, defaults to False.
+        ignore_eos_token_id(bool): Whether to ignore the EOS token id and continue generating tokens when encountering the EOS token id.
```

Comment:
`ignore_eos_token` and `ignore_eos` are more coherent to your goal.

Reply:
Done